### ISSUE TITLE
Several ids do not dereference

### ISSUE THREAD

**Issue by bpow** [clingen-data-model/clingen-interpretation#96]
Not sure how this happened, probably a leftover from renaming, but Cond070 doesn't exist. Perhaps MendCond070, but I'm not sure that it shouldn't have been renumbered as well.

**Comment by bpow** [comment]
Changing the title of this issue with some additional things to fix:

I started looking through the example data, using the results of the "reformat_examples.rb" script. calling 'by_type' on this object reveals some ids that are seen as values of attributes but do not dereference to an object (so they are not identified as having a certain type). Currently, these are:

["MendCond074", "MendCond075", "Cond070", "Ind007", "Ind008", "Ind009", "Ind010", "Ind011", "Ind012", "Ind03", "Ind030", "MendCond072", "MendCond073", "RegSeq009", "EXAC:NFE", "EXAC:SAS", "EXAC:AFR", "EXAC:FIN", "EXAC:EAS", "CG-penetrance-type:incomplete", "CG-evidence-strength:bm"]

The "MendCond" are items that have some attributes in the _MendelianConditionAttribute sheet but did not have rows in the MendelianCondition sheet. I've already fixed those.

It looks like the "Ind" should just be changed to "Indiv" prefixes.

Several of the others relate to value sets that are probably in-process.

I should probably add a warning to the site building scripts when this happens so we can more easily spot it.
